python/ray/tune/tests/test_tune_save_restore.py

```diff
@@ -83,49 +80,49 @@ def _train(self,
     def testSimpleRestore(self):
         # test base functionality of restore=<path_to_checkpoint>
         exp_name = self.prefix + "SimpleRestore"
-        local_dir = "./test_simple_restore"
-        absolute_local_dir = os.path.abspath(local_dir)
+        local_dir = "test_simple_restore"
+        absolute_local_dir = os.path.abspath(
+            os.path.join(tempfile.gettempdir(), local_dir))
+        self.absolute_local_dir = absolute_local_dir
```

Comment:
Forgot to set these before so tearDown can work properly

Reply:
Nice!